pkg/ebpftracer/c/tracee.bpf.c

```diff
@@ -6806,7 +6983,8 @@ int oom_mark_victim(struct bpf_raw_tracepoint_args *ctx)
 {
     __u32 pid = ctx->args[0];
 
-    bpf_map_update_elem(&oom_info, &pid, &pid, BPF_ANY);
+    u32 one = 1;
+    bpf_map_update_elem(&oom_info, &pid, &one, BPF_ANY);
```

Comment:
Not the biggest deal, but what is the difference of just using `pid`? Both are 32bit integers.